injected/integration-test/helpers/harness.js

```diff
@@ -17,6 +18,7 @@ export function testContextForExtension(test) {
             const tmpDirPrefix = join(tmpdir(), DATA_DIR_PREFIX);
             const dataDir = mkdtempSync(tmpDirPrefix);
             const browserTypes = { chromium, firefox };
+            const cleanupGlobals = polyfillProcessGlobals();
```

Comment:
Now this polyfil is created + cleaned up as part of the extensions lifecycle, since it was the only user